components/object-property-list/object-property-list.js

```diff
@@ -21,7 +32,10 @@ class ObjectPropertyList extends LitElement {
 	}
 
 	render() {
-		return html`<slot class="d2l-body-small"></slot>`;
+
+		return !this.skeleton || !this.skeletonCount ?
```

Comment:
I was sorta thinking these items would be rendered in the slot as default content.

```javascript
const slotCotnent = (this.skeleton && this.skeletonCount > 0 ? [...Array()].map(...) : nothing);

return html`<slot class="d2l-body-small">${slotContent}</slot>`;
```

Reply:
Looks like the slot overwrites whatever gets passed into `skeletons`.

Reply:
Any elements provided in the slot by the consumer would override the default content of the slot. Is that what you mean?

Reply:
That's right